packages/e2e/package.json

```diff
@@ -5,7 +5,9 @@
   "private": true,
   "scripts": {
     "dev": "TZ=UTC cypress open",
-    "test": "TZ=UTC cypress run --spec **/*.features"
+    "test": "TZ=UTC cypress run --spec features/ui/components/authenticator/*.features",
```

Comment:
Leaving this for now since I can't get the github action to run with the new `test.yml` changes. Will follow up with a PR to remove completely.